src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -5555,21 +5573,30 @@ namespace SimAirServingZones {
                                                                  ZoneSizing(CurOverallSimDay, CtrlZoneNum).HeatFlowSeqNoOA(TimeStepInDay));
                             SysSizing(CurOverallSimDay, AirLoopNum).HeatFlowSeq(TimeStepInDay) +=
                                 adjHeatFlowSeq / (1.0 + TermUnitSizing(TermUnitSizingIndex).InducRat);
+                            // sum up the zone heating load to be met by this system for this time step
+                            SysSizing(CurOverallSimDay, AirLoopNum).SumZoneHeatLoadSeq(TimeStepInDay) +=
+                                ZoneSizing(CurOverallSimDay, CtrlZoneNum).HeatLoadSeq(TimeStepInDay) /
+                                (1.0 + TermUnitSizing(TermUnitSizingIndex).InducRat);
```

Comment:
Line 5576 of SimAirServingZones:

@EnergyArchmage has bee struggling for a while now to figure out why some sizing results for heating were always 0. He found that these few calculations were missing. Lines 5576 - 6000. I have swept those changes in here to correct this problem. I can move these corrections to another branch if anyone has a problem with fixing this here. These in itself will probably cause diff's, as will my changes on this branch.